CauldronMods/Controller/Heroes/LadyOfTheWood/Cards/FallCardController.cs

```diff
@@ -14,31 +14,27 @@ public override void AddTriggers()
 		{
 			//Whenever LadyOfTheWood deals lightning damage to a target, reduce damage dealt by that target by 1 until the start of your next turn.
 			Func<DealDamageAction,bool> criteria = (DealDamageAction dd) => dd.DamageSource != null && dd.DamageSource.IsSameCard(base.CharacterCard) && dd.DamageType == DamageType.Lightning;
-			base.AddTrigger<DealDamageAction>(criteria, new Func<DealDamageAction, IEnumerator>(this.ReduceDamageResponse), new TriggerType[]
-			{
-				TriggerType.ReduceDamage
-			}, TriggerTiming.After);
+			AddTrigger(criteria, AddReduceDamageResponse, TriggerType.AddStatusEffectToDamage, TriggerTiming.Before);
 		}
 
-		private IEnumerator ReduceDamageResponse(DealDamageAction dd)
+		private IEnumerator AddReduceDamageResponse(DealDamageAction dd)
 		{
-			//Reduce damage dealt by that target by 1 until the start of your next turn.
-			if (dd.DidDealDamage)
+			Func<DealDamageAction, IEnumerator> statusEffectResponse = delegate (DealDamageAction dd2)
 			{
-				ReduceDamageStatusEffect reduceDamageStatusEffect = new ReduceDamageStatusEffect(1);
-				reduceDamageStatusEffect.SourceCriteria.IsSpecificCard = dd.Target;
-				reduceDamageStatusEffect.UntilStartOfNextTurn(base.TurnTaker);
-				IEnumerator coroutine = base.AddStatusEffect(reduceDamageStatusEffect);
-				if (base.UseUnityCoroutines)
-				{
-					yield return base.GameController.StartCoroutine(coroutine);
-				}
-				else
+				if (dd2.DidDealDamage)
```

Comment:
Check that the damage type is Lightning here, instead of in the initial trigger criteria.